App.js

```diff
@@ -1,106 +1,19 @@
 import React from 'react';
-import {StyleSheet, Text, View, Image, TouchableOpacity} from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import HomeScreen from './screens/HomeScreen';
+import LoginSignUp from './screens/LoginSignUp';
 
-const App = () => {
+const Stack = createNativeStackNavigator();
+
+function App() {
   return (
-    <View style={styles.homePage}>
-      <View style={styles.logoContainer}>
-        <Image source={require('./assets/logo/logo.png')} />
-      </View>
-      <View style={styles.titleLineContainer}>
-        <Text style={styles.title}>Food for Everyone</Text>
-      </View>
-      <View style={styles.imagesContainer}>
-        <View style={styles.frontImageBoundary}>
-          <Image
-            style={styles.frontImage}
-            resizeMode="contain"
-            source={require('./assets/images/animeLady.png')}
-          />
-        </View>
-        <View style={styles.backImageBoundary}>
-          <Image
-            style={styles.backImage}
-            source={require('./assets/images/animeBoy.png')}
-            resizeMode="contain"
-          />
-        </View>
-      </View>
-      <TouchableOpacity style={styles.button} onPress={() => {}}>
-        <View style={styles.buttonContainer}>
-          <Text style={[styles.buttonTitle, styles.textCenter]}>
-            Get Started
-          </Text>
-        </View>
-      </TouchableOpacity>
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator screenOptions={{headerShown: false}}>
+        <Stack.Screen name="Home" component={HomeScreen}></Stack.Screen>
+        <Stack.Screen name="LogIn" component={LoginSignUp}></Stack.Screen>
```

Comment:
Keep the screen names consistent and try to follow one pattern.
Eg. could be short form of component names like "home-screen"/ "homeScreen", "login-signup"/ "loginSignup" or could be just component name as strings.